src/Collections-Sequenceable/OrderedDictionary.class.st

```diff
@@ -171,44 +171,22 @@ OrderedDictionary >> at: aKey [
 
 { #category : #'nested dictionaries' }
 OrderedDictionary >> at: firstKey at: secondKey [
-	"Return the object stored in the second dictionary at secondKey. The second dictionary is accessed via the key firstKey."
```

Comment:
Could you keep the comment and the executable comment.

Reply:
Hi @Ducasse, thanks for your feedback (sorry for my late reply). None of the methods in OrderedDictionary that delegate to the dictionary instance variable have these comments, so I chose to remove them for consistency.
As I see it, they are implementation comments and, since the new implementation delegates to an instance of  dictionary, I feel that the comments should not be retained.

Please take a look at methods like #at:ifAbsent:, #includes:, #keyAtValue: etc. None of them have these comments, while their implementation in Dictionary does.

What do you think?